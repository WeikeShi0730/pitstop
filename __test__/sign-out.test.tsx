import { render, RenderResult } from "@testing-library/react";
import SignOut from "../components/sign-out.component";

let documentBody: RenderResult;
describe("<SignOut />", () => {
  it("Test <SignOut />", () => {
    documentBody = render(<SignOut />);
    expect(documentBody.getByText("Sign out")).toBeInTheDocument();

    const { baseElement } = documentBody;
    expect(baseElement).toMatchSnapshot();
  });
});
