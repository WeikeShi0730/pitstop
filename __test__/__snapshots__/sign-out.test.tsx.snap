// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignOut /> Test <SignOut /> 1`] = `
<body>
  <div>
    <a
      class="font-light w-full text-left text-sm md:text-base"
    >
      Sign out
    </a>
  </div>
</body>
`;
